src/Services/MASA.PM.Service.Admin/Infrastructure/Entities/Cluster.cs

```diff
@@ -15,5 +15,16 @@ public class Cluster : BaseEntity<int, Guid>
         [Required(ErrorMessage = "Cluster description is required")]
         [StringLength(255, ErrorMessage = "Cluster description length must be less than 255")]
         public string Description { get; set; } = "";
+
+        public Cluster()
```

Comment:
Maybe private enough.

Reply:
must be public, because use in other code